DOCS-2300 Add TOTP Documentation (#11277)
DataDog/documentation: content/en/synthetics/browser_tests/actions.md

#### Global variable

> Pick any global variables that was defined through [Synthetic Monitoring Settings][7].

Reviewer: Grammar who? 🙈
```suggestion
Pick any global variables defined in [Synthetic Monitoring Settings][7].
```

#### Global variable - MFA

> Pick any global variables that was defined through [Synthetic Monitoring Settings][7].

Reviewer: ```suggestion
Pick any global variables defined in [Synthetic Monitoring Settings][7].
```

> This dedicated type of global variable is designed to support multi factor authentication (MFA) processes through the [generation of time-based one time passwords (TOTP)][14].

Reviewer: ```suggestion
This type of global variable supports multi-factor authentication (MFA) processes by generating [time-based one time passwords (TOTP)][14].
```

Reviewer 2: ```suggestion
MFA global variables allows you to test your MFA modules by generating time-based one time passwords (TOTP).

Note: You can find out more on how to test downloads on [this dedicated guide][14].
```

#### Global variable

Pick any global variables defined in [Synthetic Monitoring Settings][7].

#### Global variable - MFA

Pick any MFA global variables defined in [Synthetic Monitoring Settings][7].

> This type of global variable stores time-based one time passwords (TOTPs) secret keys. Enabling you to test your MFA modules and your entire MFA-protected workflows.

Reviewer: ```suggestion

This type of global variable stores time-based one time password (TOTP) secret keys, allowing you to test your MFA modules and MFA-protected workflows.
```

> **Note:** You can find out more on how to have your browser test go through TOTP based MFA on [this dedicated guide][14].

Author: ```suggestion
**Note:** You can find out more on how to have your browser test go through TOTP based MFA on this [dedicated guide][14].
```

> **Note:** You can find out more on how to have your browser test go through TOTP based MFA on this [dedicated guide][14].

Reviewer: ```suggestion
For more information about TOTP-based MFA in a browser test, see this [TOTP guide][14].
```
